xpk.py

```diff
@@ -1368,32 +1579,68 @@ def run_gke_node_pool_create_command(args, system) -> int:
 
   commands = []
   task_names = []
-  for node_pool_name in desired_node_pool_names:
-    if node_pool_name in existing_node_pool_names:
-      continue
-    command = (
-        'gcloud beta container node-pools create'
-        f' {node_pool_name} --node-version={args.gke_version}'
-        f' --cluster={args.cluster}'
-        f' --project={args.project} --node-locations={args.zone}'
-        f' --region={zone_to_region(args.zone)}'
-        f' --num-nodes={system.vms_per_slice}'
-        f' --machine-type={system.gce_machine_type}'
-        f' --host-maintenance-interval={args.host_maintenance_interval}'
-        f' {capacity_args}'
-        ' --scopes=storage-full,gke-default'
-        ' --enable-gvnic --max-pods-per-node 15'
+
+  device_type = args.tpu_type if args.tpu_type else args.device_type
```

Comment:
You don't need this check in this case, "system" arg will tell you the device_type.

`if system.device_type == h100_device_type`